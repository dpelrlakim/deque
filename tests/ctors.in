newDefault dq1
newInit dq1
= dq1 dq2
push_back dq2 0
newInit dq2
= dq1 dq2
newDefault dq3
push_front dq3 -2000
= dq1 dq3
pop_back dq1
= dq2 dq1
[]= dq3 0 3000
= dq1 dq3
push_back dq1 1
push_back dq1 2
push_back dq1 3
push_back dq1 4
push_back dq1 5
= dq2 dq1
at dq2 6
[] dq2 5
= dq2 dq3
= dq2 dq1
= dq3 dq2
