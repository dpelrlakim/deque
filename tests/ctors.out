dq1:
dq1: 2 3 5 7 11 13 17 19
dq1:
dq2:
dq1:
dq2: 0
dq1:
dq2: 2 3 5 7 11 13 17 19
dq1: 2 3 5 7 11 13 17 19
dq2: 2 3 5 7 11 13 17 19
dq1: 2 3 5 7 11 13 17 19
dq2: 2 3 5 7 11 13 17 19
dq3:
dq1: 2 3 5 7 11 13 17 19
dq2: 2 3 5 7 11 13 17 19
dq3: -2000
dq1: -2000
dq2: 2 3 5 7 11 13 17 19
dq3: -2000
dq1:
dq2: 2 3 5 7 11 13 17 19
dq3: -2000
dq1:
dq2:
dq3: -2000
dq1:
dq2:
dq3: 3000
dq1: 3000
dq2:
dq3: 3000
dq1: 3000 1
dq2:
dq3: 3000
dq1: 3000 1 2
dq2:
dq3: 3000
dq1: 3000 1 2 3
dq2:
dq3: 3000
dq1: 3000 1 2 3 4
dq2:
dq3: 3000
dq1: 3000 1 2 3 4 5
dq2:
dq3: 3000
dq1: 3000 1 2 3 4 5
dq2: 3000 1 2 3 4 5
dq3: 3000
Exception
dq1: 3000 1 2 3 4 5
dq2: 3000 1 2 3 4 5
dq3: 3000
5
dq1: 3000 1 2 3 4 5
dq2: 3000 1 2 3 4 5
dq3: 3000
dq1: 3000 1 2 3 4 5
dq2: 3000
dq3: 3000
dq1: 3000 1 2 3 4 5
dq2: 3000 1 2 3 4 5
dq3: 3000
dq1: 3000 1 2 3 4 5
dq2: 3000 1 2 3 4 5
dq3: 3000 1 2 3 4 5
